README.md

```diff
@@ -138,6 +138,29 @@ render json: @post, meta: { total: 10 }, meta_key: "custom_meta"
 
 `meta` will only be included in your response if you are using an Adapter that supports `root`, as JsonAPI and Json adapters, the default adapter (FlattenJson) doesn't have `root`.
 
+### Using a serializer without `render`
+
+At times, you might want to use a serializer without rendering it to the view. For those cases, you can create an instance of `ActiveModel::SerializableResource` with
+the resource you want to be serialized and call `.serializable_hash`.
+
+```ruby
+def create
```

Comment:
for the sake of readme length, do you think this should be simplified?

having context is nice though.

@joaomdmoura, what are your thoughts?